### ISSUE TITLE
failed to build the project

### ISSUE THREAD

**Issue by Jack-samu** [v2rayA/v2raya.github.io#251]
## Description

failed to yarn my project and found the zip was closed or something else as a result.

### Expected result

yarn the project successfully

### Actual result

I downloaded the project and build it with the 'yarn' last night , but I kept failing to build my it and until I found it.

![image](https://user-images.githubusercontent.com/58771827/232193885-9b13e952-e818-41d5-9398-a8872528d8c3.png)

I' m not sure whether it is a bug, but I want some help from you as a newbie in this field, thank you. Also, the problem in the building project is this:

![image](https://user-images.githubusercontent.com/58771827/232194050-df2da310-8c1f-4966-9294-13b34268383a.png)

As I came to the releases of the hugo and start download the condition which is the first screenshot happened.

### Environment

win11 nodejs with yarn

**Comment by MarksonHon** [comment]
We are Not part of hugo, sorry

**Comment by Jack-samu** [comment]
No, it was happened in building the v2raya-doc project, it' s drived by the hugo, right? I just want to ask the substitute.
